ffmpeg/ffmpeg.go

```diff
@@ -171,6 +172,28 @@ func Transcode3(input *TranscodeOptionsIn, ps []TranscodeOptions) (*TranscodeRes
 	return t.Transcode(input, ps)
 }
 
+// Force copy-transcoding for audio-only segments
+func BypassTranscoding(input *TranscodeOptionsIn, ps []TranscodeOptions) (*TranscodeResults, error) {
+	glog.Errorf("WARNING: Input %s has audio & video streams but no video frames. Bypassing the transcoder.\n", input.Fname)
```

Comment:
Seems like `glog.Warningf()` would be a good fit here.

Reply:
> Seems like glog.Warningf() would be a good fit here.
Ah had no idea this existed, perfect!

> Side note: Do we want the extra newline at the end? Just wondering if it was purposefully added for visual reasons.
Bad habit from C's printf function, will remove thx! I see glog will add a newline if not present, and leave it as-is if present https://godoc.org/github.com/golang/glog#Warningf